config.py.example

```diff
@@ -101,3 +102,20 @@ else:
 # your own access token. See
 # help.github.com/articles/creating-an-access-token-for-command-line-use
 OAUTH_TOKEN = os.environ.get('OAUTH_TOKEN') or ""
+
+# Status categories used in the project
+# 'new', 'needsdiagnosis', 'needscontact', 'contactready' , 'sitewait', 'close'
+# Creating the model
+Category = namedtuple('Category', ['name', 'dataAttribute', 'label'])
+CATEGORIES = []
+cat_labels = [('new', 'new', 'New Issues'),
+              ('needsDiagnosis', 'needsdiagnosis', 'Needs Diagnosis'),
+              ('needsContact', 'needscontact', 'Needs Contact'),
+              ('ready', 'contactready', 'Ready for Outreach'),
+              ('sitewait', 'sitewait', 'Site Contacted'),
+              ('close', 'closed', 'Closed')]
+# populating the categories for issues status
+for cat_label in cat_labels:
+    CATEGORIES.append(Category(name=cat_label[0],
+                               dataAttribute=cat_label[1],
+                               label=cat_label[2]))
```

Comment:
I don't have preferences apart of the possibility of evolving them. Right now the dict proposed is short. 
The namedtuple was chosen according to what magsout was trying to do in the template section. And calling the variable with a `Object.value` syntax.